# Fix WebKit compilation for arm64. Use 64bits for all platforms
--- wkhtmltopdf-0.12.5/qt/src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h	2018-05-29 06:56:52.000000000 +0000
+++ patches-0.12.5/Platform.h	2025-06-27 00:46:10.686241352 +0000
@@ -996,6 +996,7 @@
 #define ENABLE_FULLSCREEN_API 0
 #endif

+#define WTF_USE_JSVALUE64 1
 #if !defined(WTF_USE_JSVALUE64) && !defined(WTF_USE_JSVALUE32_64)
 #if (CPU(X86_64) && (OS(UNIX) || OS(WINDOWS))) \
     || (CPU(IA64) && !CPU(IA64_32)) \
